docs/automated-process.md

```diff
@@ -11,6 +11,34 @@ release process. The automated processes uses GitHub Actions to automate the
     `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
     specify.
 
+## Setup for the release
+
+### Credentials
+
+- Access to https://github.com/ansible-community/ansible-build-data repo
+- Access to https://github.com/ansible/ansible repo
+- Access to Ansible Release Management working group (in Github).
```

Comment:
I think "membership" is probably a better word. Also, Release Management WG members should join the forum group.